inductiva/resources/machines.py

```diff
@@ -15,6 +27,7 @@ class MachineGroup(machines_base.BaseMachineGroup):
     def __init__(
         self,
         machine_type: str,
+        provider: Literal["GCP", "ICE"] = "GCP",
```

Comment:
Why not lower case for these defaults as well

Reply:
Because, the backend established it like this...